### DISCUSSION TITLE
布局相关建议

### DISCUSSION THREAD

**Discussion by lc6464** [YerongAI/Office-Tool#544] [Ideas]
`9.0.2.10` 版本“设置”和“关于”按钮均在左下角，页面（想不出什么更好的表述方式）却出现在右侧，是否有点不协调？我觉得可以考虑移到左边。

另外，建议在“关于”页面为版本号添加点击复制功能，现在看起来似乎是检查更新？我觉得要能让用户手动选择版本号以进行复制比较合理，现在是不可选择的。

**Comment by YerongAI** [comment]
1. 左侧会覆盖导航栏，这是不合理的。
2. 版本号这么短也要复制？手敲不行吗？以及你觉得是检查更新多还是复制版本号多？

**Reply by lc6464** [reply]
一键复制无所谓了，主要是现在版本号不能选择，![image](https://user-images.githubusercontent.com/64722907/178986472-15398c92-3e30-4716-b372-d778266aa845.png)

**Reply by YerongAI** [reply]
> 一键复制无所谓了，主要是现在版本号不能选择，![image](https://user-images.githubusercontent.com/64722907/178986472-15398c92-3e30-4716-b372-d778266aa845.png)

你选择版本号要干啥呢？复制版本号又要干啥呢？

**Reply by lc6464** [reply]
> 你选择版本号要干啥呢？复制版本号又要干啥呢？

也许在什么时候发 Issue 或者 Discussion（比如说这条）的时候能用上？

**Reply by YerongAI** [reply]
> > 你选择版本号要干啥呢？复制版本号又要干啥呢？
> 
> 也许在什么时候发 Issue 或者 Discussion（比如说这条）的时候能用上？

那就手敲啊，那么短的版本号又不是敲不动

**Reply by lc6464** [reply]
> 那就手敲啊，那么短的版本号又不是敲不动

~~（眼瞎👀）~~
手敲可以是可以，允许选择的话更方便嘛。
这个在以后更新的时候顺手改一下也不难吧？

**Reply by YerongAI** [reply]
> > 那就手敲啊，那么短的版本号又不是敲不动
> 
> ~（眼瞎👀）~ 手敲可以是可以，允许选择的话更方便嘛。 这个在以后更新的时候顺手改一下也不难吧？

难

**Reply by lc6464** [reply]
> 难

那好吧。
